chart/keda/README.md

```diff
@@ -45,6 +45,21 @@ $ helm install kedacore/keda-edge --devel --set logLevel=debug --namespace keda
 func kubernetes install --namespace keda
 ```
 
+## Configuration
+
+| Parameter                         | Description                         | Default              |
+|:----------------------------------|:------------------------------------|:---------------------|
+| `image.repository`                | Repository which provides the image | `kedacore/keda`      |
+| `image.tag`                       | Tag of image to use | `lastest`            |
+| `image.pullPolicy`                | Policy to pull image | `Always`            |
+| `replicaCount`                    | Amount of replicas to run | `1`            |
+| `customResourceDefinition.create` | Indication to whether or not to create the custom resource definition | `true`            |
+| `rbac.create`                     | Indication to whether or not to use role-based access control | `true`            |
+| `serviceAccount.create`           | Indication to whether or not to a serivce account should be used | `true`            |
+| `serviceAccount.name`             | Name of the service account to use | ``            |
+| `logLevel`                        | Granularity of logs to use | `info`          |
```

Comment:
```suggestion
| `logLevel`                        | Granularity of logs to use for the metrics adapter | `info`          |
```

Would this description make sense @ahmelsayed ? Now it's a bit vague.

Reply:
@tomkerkhove the logLevel is the granularity of logs to use for both the metrics adapter and the scale controller- so all Keda.
gLogLevel determines the log of additional components we use- the MetricAdapter boilerplate etc.

So a comment describing that would be useful. Users can start out with setting logLevel =Debug to get more KEDA specific logs. They can make the glogLevel more verbose to get the REST calls made by HPA etc to the metrics adapter. HTH.

Reply:
Thanks for elaborating on this, so basically:
- `logLevel` controls KEDA-specific logs (controller + metrics adapter
- `glogLevel` controls metric adapter core metrics, not specific to KEDA.

Is that correct summary? If so, I'll suggest a better description.

Reply:
@tomkerkhove That is correct. And setting the glog verbose level will also pull in the glogs of other dependencies we have (though I primarily see the REST API calls).

Reply:
I've made the changes accordingly, feel free to let me know what you think @Aarthisk